tests/networks/nets/test_quicknat.py

```
@@ -36,6 +46,38 @@
 ]
 
 
+class TestQuicknatCore(unittest.TestCase):
+    """Test QuickNAT paths that do not require optional dependencies."""
+
+    @parameterized.expand(["cat", "add", "mul"])
+    def test_skip_connection_modes_preserve_indices(self, mode):
+        """Verify each fusion mode and preservation of pooling indices."""
+        skip = SkipConnectionWithIdx(_DoubleWithIdx(), mode=mode)
+        input = torch.tensor([[[[2.0]]]])
```

Ruff (0.15.21)

[error] Variable `input` is shadowing a Python builtin

(A001)